pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -3024,6 +3024,25 @@ self: super: {
       sha256 = "sha256-IxtyHFKygbrUK7JTAytWBZVHh+M1xQTv2IPCG3mjTGE=";
       stripLen = 1;
     }) super.yesod-core;
+
+  bsb-http-chunked = lib.pipe super.bsb-http-chunked [
+    # Last released in 2018
+    # https://github.com/sjakobi/bsb-http-chunked/issues/38
+    # https://github.com/sjakobi/bsb-http-chunked/issues/45
+    (overrideSrc {
+      src = pkgs.fetchFromGitHub {
+        owner = "sjakobi";
+        repo = "bsb-http-chunked";
+        rev = "c0ecd72fe2beb1cf7de9340cc8b4a31045460532";
+        hash = "sha256-+UDxfywXPjxPuFupcB8veyMYWVQCKha64me9HADtFGg=";
+      };
+    })
```

Comment:
I assume you are saying there is no way to do this as a patch?

Reply:
Well, there's been quite a few commits since the 2018 release
https://github.com/sjakobi/bsb-http-chunked/commits/master/
I guess one could manually make a patch containing all that, but is it much of an improvement?

Reply:
Would look like this
https://github.com/sjakobi/bsb-http-chunked/compare/v0.0.0.4...master

Reply:
> is it much of an improvement?

Ah, I guess doing this via patch would warn us as soon as the hackage release had progressed?
Whereas the blanket override will stagnate a lot longer?